setup/azurecmdfiles.wxi

```diff
@@ -336,41 +336,6 @@
-                        </Component>
-                        <Component Id="cmp8C5CED7D44B67AD12A15B08F1DF3E376" Guid="*">
-                            <File Id="fil5FD33269FC93C30886683D163361F953" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\ConvertTo-AzureRmVhd.psm1" />
-                        </Component>
-                        <Component Id="cmp1800E5A3128A485B6343BFFB8DFA8F75" Guid="*">
-                            <File Id="fil232EDE5D3B7E7DD2A29FDD1E031980AB" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Microsoft.Azure.Commands.Compute.format.generated.ps1xml" />
-                        </Component>
-                        <Component Id="cmp9550ED84E4EB33AB8AE4DA0A0BEF650C" Guid="*">
-                            <File Id="filEBE4D010E62434598654429A998DDECE" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Microsoft.Azure.Commands.Compute.format.ps1xml" />
-                        </Component>
-                        <Component Id="cmpFC32857975951E6B04E23780525895ED" Guid="*">
-                            <File Id="fil94A3AA801F2A54E38DDB8805B2C9BE0D" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Microsoft.Azure.Commands.Compute.ManagedService.dll" />
-                        </Component>
-                        <Component Id="cmpA87B9B4AA3D02C3ED31B9A0696556795" Guid="*">
-                            <File Id="fil43384C949987880EA81F65039C422C05" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Microsoft.Azure.Commands.Compute.Types.ps1xml" />
-                        </Component>
-                        <Directory Id="dir0F51DEA6EAFCB2EDAD3EC33DF2E2DE3F" Name="Generated">
-                            <Component Id="cmp49D442B0CBF1C941D483DAA9E8883CA4" Guid="*">
-                                <File Id="fil86C6139B21C1A398ACEA482B49A078D3" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Generated\Microsoft.Azure.Commands.Compute.Automation.format.generated.ps1xml" />
-                            </Component>
-                        </Directory>
-                        <Directory Id="dirD472965593F6562EA7FBCE3FB9141CB3" Name="Tools">
-                            <Component Id="cmp2B5B03A1A365F2E65CE7D11B56AD353B" Guid="*">
-                                <File Id="filE04C56E5EDD05F7C11DBA461E03A8B95" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService\Tools\ConvertTo-AzureRmVhd.ps1" />
-                            </Component>
-                        </Directory>
                     </Directory>
```

Comment:
@MiYanni looks like this `</Directory>` tag didn't get deleted, which is causing the failure in the build

Reply:
@cormacpayne I think you commented at the exact same time I pushed the fix.